docs/docsite/rst/porting_guides/porting_guide_8.rst

```diff
@@ -94,6 +94,47 @@ Networking
 
 No notable changes
 
+Porting Guide for v8.3.0
+========================
+
+Added Collections
+-----------------
+
+- telekom_mms.icinga_director (version 1.34.1)
+
+Major Changes
+-------------
+
+fortinet.fortios
+~~~~~~~~~~~~~~~~
+
+- Add readthedocs.yaml file.
+- Update Q&A regarding setting up FortiToken multi-factor authentication;
+
+Deprecated Features
+-------------------
+
+- The collection ``t_systems_mms.icinga_director`` has been renamed to ``telekom_mms.icinga_director``. For now both collections are included in Ansible. The content in ``t_systems_mms.icinga_director`` will be replaced with deprecated redirects to the new collection in Ansible 9.0.0, and these redirects will eventually be removed from Ansible. Please update your FQCNs for ``t_systems_mms.icinga_director``.
+- The netapp.azure collection is considered unmaintained and will be removed from Ansible 10 if no one starts maintaining it again before Ansible 10. See `the removal process for details on how this works <https://github.com/ansible-collections/overview/blob/main/removal_from_ansible.rst#cancelling-removal-of-an-unmaintained-collection>`__ (https://github.com/ansible-community/community-topics/issues/234).
```

Comment:
fwiw I think most of these added links won't work. They all seem to have a double underscore. I'm testing the devel branch now. Won't hold up the release but we may need to open an issue on this collection to get them fixed if they are broken.

Reply:
ok these actually work. TIL.

Reply:
In fact they are very useful to reduce the amount of warnings Sphinx or rstcheck prints ;-)